### PR TITLE
[SYCL] Removed the remained handler::codeplay_host_task

### PR DESCRIPTION
**Pull request by maximdimakov**
Signed-off-by: mdimakov maxim.dimakov@intel.com

### CODE REVIEW

File: SYCL/Basic/accessor/Inputs/host_task_accessor.cpp [hunk 1]
```diff
@@ -54,9 +54,6 @@ int main() {
 
 #if defined(accessor_new_api_test) || defined(buffer_new_api_test)
```
Comment (dm-vodopyanov) [inline SYCL/Basic/accessor/Inputs/host_task_accessor.cpp:55]: Is this if-statement needed after the removal of codeplay_host_task?

File: SYCL/Basic/accessor/Inputs/host_task_accessor.cpp [hunk 2]
```diff
@@ -115,16 +113,14 @@ int main() {
...
```
Comment (dm-vodopyanov) [inline SYCL/Basic/accessor/Inputs/host_task_accessor.cpp:114]: And here

File: SYCL/HostInteropTask/interop-task.cpp [hunk 3]
```diff
@@ -161,14 +158,14 @@ template <bool UseSYCL2020HostTask> void test2_1(queue &Q) {
...
-template <bool UseSYCL2020HostTask> void test3(queue &Q) {
+void test3(queue &Q) {
   // Want some large buffer for operation to take long
   buffer<int, 1> Buffer{BUFFER_SIZE * 128};
 
   event Event = Q.submit([&](handler &CGH) {
     auto Acc1 = Buffer.get_access<mode::write>(CGH);
 
-    CGH.parallel_for<NameGen<UseSYCL2020HostTask, class Init3>>(
+    CGH.parallel_for<NameGen<true, class Init3>>(
```
Comment (dm-vodopyanov) [inline SYCL/HostInteropTask/interop-task.cpp]: Is `NameGen` still needed here?

File: SYCL/HostInteropTask/interop-task.cpp [hunk 4]
```diff
@@ -181,40 +178,33 @@ template <bool UseSYCL2020HostTask> void test3(queue &Q) {
...
   buffer<int, 1> Buffer1{BUFFER_SIZE};
   buffer<int, 1> Buffer2{BUFFER_SIZE};
 
   Q.submit([&](handler &CGH) {
     auto Acc = Buffer1.template get_access<mode::write>(CGH);
 
     auto Kernel = [=](item<1> Id) { Acc[Id] = 123; };
-    CGH.parallel_for<NameGen<UseSYCL2020HostTask, class Test5Init>>(
-        Acc.get_count(), Kernel);
+    CGH.parallel_for<NameGen<true, class Test5Init>>(Acc.get_count(), Kernel);
```
Comment (dm-vodopyanov) [inline SYCL/HostInteropTask/interop-task.cpp]: And here

File: SYCL/Basic/accessor/Inputs/host_task_accessor.cpp (hunk 1 above)
Comment (maximdimakov) [reply SYCL/Basic/accessor/Inputs/host_task_accessor.cpp:55]: All of the accessors are defined under #ifdef so they are not available without this check

File: SYCL/HostInteropTask/interop-task.cpp (hunk 4 above)
Comment (maximdimakov) [reply SYCL/HostInteropTask/interop-task.cpp]: Removed

**Review by dm-vodopyanov — DISMISSED** [review]
LGTM

**Review by dm-vodopyanov — APPROVED** [review]

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**